sbctl.go

```diff
@@ -214,14 +230,26 @@ func CreateBundle(bundle Bundle) {
 	defer os.Remove(tmpFile.Name())
 	bundle.Initramfs = tmpFile.Name()
 
-	GenerateBundle(&bundle)
+	return GenerateBundle(&bundle)
 }
 
-func GenerateAllBundles() {
+func GenerateAllBundles() error {
 	msg.Println("Generating EFI bundles....")
 	bundles := ReadBundleDatabase(BundleDBPath)
+	out := true
 	for _, bundle := range bundles {
-		CreateBundle(*bundle)
+		if !CreateBundle(*bundle) {
+			out = false
+			err2.Printf("failed to write bundle %s!", bundle.Output)
+		}
+	}
+
+	if out {
+		return nil
+	} else {
+		msg := "Error generating EFI bundles"
+		err2.Println(msg)
+		return errors.New(msg)
```

Comment:
`if !out` then you can throw the `return nil` at the end of the function. Might look more natural?

Reply:
It does, thanks!